spark-atlas-connector/src/main/scala/org/apache/spark/sql/kafka010/atlas/KafkaHarvester.scala

```diff
@@ -50,11 +50,16 @@ object KafkaHarvester extends AtlasEntityUtils with Logging {
     // destination topic
     var destTopic = None: Option[String]
     try {
-      destTopic = node.getClass.getMethod("topic").invoke(node).asInstanceOf[Option[String]]
+      val topicField = node.getClass.getDeclaredField("topic")
+      topicField.setAccessible(true)
```

Comment:
Besides the error handling at line 75, is this line only effective change, `topicField.setAccessible(true)`?

Reply:
yes, we can not get the topic information by the provided approach. I do not know the reason, but it seems the topic is a declaredField rather than a method.

Reply:
Actually, this seems to be directly related to @weiqingy 's internal patch you merged today.
@weiqingy . Could you check if we can revert your internal commit?

Reply:
The internal commit to Spark repo also modified the right to access "writer" information:                     https://github.com/hortonworks-spark/spark-atlas-connector/blob/master/spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/SparkExecutionPlanTracker.scala#L134

It's wired. If SAC is able to get "writer" info, why it is unable to get "topic" info?? They are the same issue...

Reply:
Could you share you cluster?? @merlintang I need to test it first.

Reply:
sure, weiqing, thanks for your kind help. let us look at what is going here.

Reply:
@merlintang , did you share the cluster to @weiqingy ?